frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -31,29 +29,8 @@ export const PoolResourceComponent: React.FC<ProvisionerProps> = ({ onParamChang
     cephClusterResource,
   );
 
-  const [scObjList, scloaded, scloadError] = useK8sWatchResource<StorageClassResourceKind[]>(
-    scResource,
-  );
-
   const [isOpen, setOpen] = React.useState(false);
   const [poolName, setPoolName] = React.useState('');
-  const [usedPool, setUsedPool] = React.useState<string[]>(null);
-
-  React.useEffect(() => {
-    if (scloaded && !scloadError) {
-      const usedPoolArray = _.reduce(
-        scObjList,
-        (res, value) => {
-          if (value.provisioner === OCS_PROVISIONER.BLOCK) {
-            res.push(value?.['parameters']?.pool);
-          }
-          return res;
-        },
-        [],
-      );
-      setUsedPool(usedPoolArray);
-    }
-  }, [scObjList, scloadError, scloaded]);
```

Comment:
The BZ says to show used + unused pools as well.
https://bugzilla.redhat.com/show_bug.cgi?id=1878075#c0
Why to remove then ?

Reply:
The list of storageClass was fetched just to check for used pools, since we dont need that information now, we can remove this code.

Reply:
The information I was looking for about the removal of `usedPools` when it was asked in BZ.
Tallying with the existing code I found we have been using this setting to actually disable to show unused pools. Its cleared now.